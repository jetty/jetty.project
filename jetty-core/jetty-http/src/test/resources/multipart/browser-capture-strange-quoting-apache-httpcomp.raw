--z5xWs05oeiE0TAdFlrrlAX5RSgHrHzVcgskrru
Content-Disposition: form-data; name="and \"I\" quote"
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Value 1
--z5xWs05oeiE0TAdFlrrlAX5RSgHrHzVcgskrru
Content-Disposition: form-data; name="and+%22I%22+quote"
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Value 2
--z5xWs05oeiE0TAdFlrrlAX5RSgHrHzVcgskrru
Content-Disposition: form-data; name="value\"; what=\"whoa\""
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Value 3
--z5xWs05oeiE0TAdFlrrlAX5RSgHrHzVcgskrru
Content-Disposition: form-data; name="other\";	what=\"Something\""
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Value 4
--z5xWs05oeiE0TAdFlrrlAX5RSgHrHzVcgskrru--
