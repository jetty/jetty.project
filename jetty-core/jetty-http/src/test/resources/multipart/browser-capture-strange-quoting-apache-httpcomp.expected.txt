Request-Header|Accept-Encoding|gzip,deflate
Request-Header|Connection|keep-alive
Request-Header|Content-Length|798
Request-Header|Content-Type|multipart/form-data; boundary=z5xWs05oeiE0TAdFlrrlAX5RSgHrHzVcgskrru
Request-Header|Host|localhost:9090
Request-Header|User-Agent|Apache-HttpClient/4.5.5 (Java/1.8.0_162)
Request-Header|X-BrowserId|apache-httpcomp
Parts-Count|4
Part-ContainsContents|and "I" quote|Value 1
Part-ContainsContents|and+%22I%22+quote|Value 2
Part-ContainsContents|value"; what="whoa"|Value 3
